### ISSUE TITLE
feature request: an ordinary way to terminate the app?

### ISSUE THREAD

**Issue by muellerto** [KillerInk/FreeDcam#203]
I'm not sure if I'm right but ...

... is there really no better way to terminate the app than moving it into background using the [->] button in the upper right corner and then using the now visible square button of the OS to get it back into foreground and closing it there? When I'm on a day-long mountain hiking tour I must do this clumsy procedure 30 times or more.

Why can't the [->] button be an ordinary  [x] button and terminate the app reliably? I would prefer this.

**Comment by KillerInk** [comment]
thats also solved. that problem was freedcam locked the event and did not pass it further.

**Comment by muellerto** [comment]
Yeah, but where to get it? Mine is from F-Droid, and it's the latest available there. The /e/-App store has also no other version, 4.3.65 (265).

**Comment by muellerto** [comment]
I still don't have any solution for that. If I tap on the [->] button in the upper right corner the app just disappears but is still runs in background.

And the NextGen GUI which can the user switch on makes it even worse:
![Screenshot_20221207-124948_FreeDCam-1](https://user-images.githubusercontent.com/5188366/206173359-ec53c1d5-2f88-45d4-b9c2-ff0c10edfe55.png)
How can I terminate the app here? There's no control at all.

I use 4.3.82 from F-Droid.
